config/routes.rb

```diff
@@ -1,5 +1,9 @@
 Rails.application.routes.draw do
 
+  namespace :api, defaults: { format: :json } do
+    resources :users
```

Comment:
`resources :users` is generating unused routes (see `rake routes` output to check). Try to limit route configuration to generate only those routes that are used.